device-registry/src/main/scala/org/genivi/sota/device_registry/GroupsResource.scala

```diff
@@ -38,10 +39,17 @@ class GroupsResource(namespaceExtractor: Directive1[Namespace])
     }
   }
 
+  def getDevicesInGroup(groupName: Name, namespace: Namespace): Route = {
+    complete(db.run(GroupMember.listDevicesInGroup(groupName, namespace)))
+  }
+
   val route: Route =
     (pathPrefix("device_groups") & namespaceExtractor) { ns =>
       (post & pathPrefix("from_attributes") & pathEnd) {
         entity(as[CreateGroupRequest]) { groupInfo => createGroupFromDevices(groupInfo, ns) }
+      } ~
+      (get & pathEnd & parameter('groupName.as[Name])) { groupName =>
```

Comment:
No, I think uuid for each group would be an internal thing. You would still give it the group name and get a list of device uuids back.